mon-pix/package.json

```
@@ -24,7 +24,7 @@
     "clean": "rm -rf tmp dist node_modules",
     "dev": "ember server",
     "lint": "npm run lint:js && npm run lint:hbs && npm run lint:scss",
-    "lint:fix": "npm run lint -- --fix",
+    "lint:fix": "npm run lint:js -- --fix && npm run lint:hbs -- --fix && npm run lint:scss -- --fix",
```

```suggestion
"lint:fix": "run-p 'lint:* -- --fix'"
```
A priori pas besoin d'ajouter npm-run-all puisqu'il est deja dans le .bin du parent

---

Super idée :heart: J'ouvre une PR là-dessus de suite